app/views/questionnaire.py

```diff
@@ -414,33 +403,37 @@ def _save_sign_out(routing_path, current_location, current_question, form, schem
 
         return redirect(url_for('session.get_sign_out'))
 
-    context = _get_context(routing_path, block, current_location, schema, form)
+    context = _get_context(block, current_location, schema, form)
     return _render_page(block['type'], context, current_location, schema)
 
 
+def _evaluate_skip_conditions(block_json, schema, answer_store, metadata):
+    for question in schema.get_questions_for_block(block_json):
+        if 'skip_conditions' in question:
+            skip_question = evaluate_skip_conditions(question['skip_conditions'], schema, metadata, answer_store)
+            question['skipped'] = skip_question
+            for answer in question['answers']:
+                if answer['mandatory'] and skip_question:
+                    answer['mandatory'] = False
+
+    return block_json
```

Comment:
Now we have variants, skipping questions should be removed.

Reply:
Wasn't even called by anything